variables.tf

```diff
@@ -8,49 +8,93 @@ variable "VM_NAME" {
   default = "polygon-user"
 }
 
-variable "DISK_SIZE_GB" {
+variable "BOR_DISK_SIZE_GB" {
   default = "100"
 }
 
-variable "ARCHIVE_DISK_SIZE_GB" {
+variable "ERIGON_DISK_SIZE_GB" {
   default = "100"
 }
 
-variable "IOPS" {
+variable "BOR_ARCHIVE_DISK_SIZE_GB" {
+  default = "100"
+}
+
+variable "ERIGON_ARCHIVE_DISK_SIZE_GB" {
+  default = "100"
+}
+
+variable "BOR_IOPS" {
   default = 3000
 }
 
-variable "ARCHIVE_IOPS" {
+variable "ERIGON_IOPS" {
+  default = 3000
+}
+variable "BOR_ARCHIVE_IOPS" {
   default = 3000
 }
 
-variable "VOLUME_TYPE" {
+variable "ERIGON_ARCHIVE_IOPS" {
+  default = 3000
+}
+
+variable "BOR_VOLUME_TYPE" {
+  default = "gp3"
+}
+
+variable "ERIGON_VOLUME_TYPE" {
   default = "gp3"
 }
 
-variable "ARCHIVE_VOLUME_TYPE" {
+variable "BOR_ARCHIVE_VOLUME_TYPE" {
+  default = "io1"
+}
+
+variable "ERIGON_ARCHIVE_VOLUME_TYPE" {
   default = "io1"
 }
 variable "DOCKERIZED" {
   default = "no"
 }
 
-variable "VALIDATOR_COUNT" {
+variable "BOR_VALIDATOR_COUNT" {
   default = "2"
 }
 
-variable "SENTRY_COUNT" {
+variable "ERIGON_VALIDATOR_COUNT" {
+  default = "0"
+}
+
+variable "BOR_SENTRY_COUNT" {
   default = "1"
 }
 
-variable "ARCHIVE_COUNT" {
+
+variable "ERIGON_SENTRY_COUNT" {
+  default = "0"
+}
+
+variable "BOR_ARCHIVE_COUNT" {
+  default = "0"
+}
+
+variable "ERIGON_ARCHIVE_COUNT" {
   default = "0"
 }
-variable "INSTANCE_TYPE" {
+variable "BOR_INSTANCE_TYPE" {
+  default = "t2.xlarge"
+}
+
+variable "ERIGON_INSTANCE_TYPE" {
+  default = "t2.xlarge"
```

Comment:
I think we should have r5b.large in case of erigon, generally speaking erigon nodes perform better on memory optimised instances and r5b.large is cheaper than t2.xl